packages/experimental/ActivityIndicator/src/ActivityIndicator.styling.tsx

```diff
@@ -0,0 +1,46 @@
+import { UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
+import {
+  activityIndicatorName,
+  ActivityIndicatorProps,
+  ActivityIndicatorSlotProps,
+  ActivityIndicatorTokens,
+} from './ActivityIndicator.types';
+
+export const diameterSizeMap: { [key: string]: number } = {
+  xSmall: 12,
+  small: 16,
+  medium: 24,
+  large: 32,
+  xLarge: 36,
+};
```

Comment:
@rurikoaraki Not an ask, but this may be a good candidate for a design token that is not a color to integrate from the pipeline come time. same for the line thickness below.